common/actions/ens/actionTypes/resolveDomain.ts

```diff
@@ -0,0 +1,28 @@
+import { TypeKeys } from '../constants';
+import { DomainRequest } from 'libs/ens';
+
+export interface ResolveDomainRequested {
+  type: TypeKeys.ENS_RESOLVE_DOMAIN_REQUESTED;
+  payload: { domain: string };
+}
+
+export interface ResolveDomainSucceeded {
+  type: TypeKeys.ENS_RESOLVE_DOMAIN_SUCCEEDED;
+  payload: { domain: string; domainData: DomainRequest };
+}
+
+export interface ResolveDomainCached {
+  type: TypeKeys.ENS_RESOLVE_DOMAIN_CACHED;
+  payload: { domain: string };
+}
```

Comment:
For simplicity, does this need to be a separate action? Can we just not re-use succeeded?

Reply:
Would rather have it explicitly different for readability reasons